nbs/tutorials/extensions.ipynb

```diff
@@ -0,0 +1,557 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.            if len(code) == 1:
</pre><p>this could be a one liner using ternary op</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/1228/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>